packages/aws-cdk-lib/aws-opensearchservice/README.md

```diff
@@ -349,6 +349,36 @@ const domain = new Domain(this, 'Domain', {
 });
 ```
 
+## Suppress creating CloudWatch Logs resource policy
+
+When logging is enabled for the domain, the CloudWatch Logs resource policy is created by default.
+This resource policy is necessary for logging, but since only a maximum of 10 resource policies can be created per region,
+the maximum number of resource policies may be a problem when enabling logging for several domains.
+By setting the `suppressLogsResourcePolicy` option to true, you can suppress the creation of a CloudWatch Logs resource policy.
```

Comment:
Can you also hint at the side-effects of adding this prop? What does it mean to not have a resource policy

Reply:
Right.
I added a description about this to readme and jsdoc.